include/dos_system.h

```diff
@@ -189,15 +190,21 @@ class localFile : public DOS_File {
 	{
 		return basedir;
 	}
+	std_fs::path GetPath() const
+	{
+		return path;
+	}
 	FILE* fhandle = nullptr; // todo handle this properly
 private:
-	const char *basedir;
-	long stream_pos = 0;
+	const std_fs::path path = {};
+	const char* basedir     = nullptr;
+	long stream_pos         = 0;
 	bool ftell_and_check();
 	void fseek_and_check(int whence);
 	bool fseek_to_and_check(long pos, int whence);
-	bool read_only_medium;
-	enum { NONE,READ,WRITE } last_action;
+	bool read_only_medium                  = false;
+	enum { NONE, READ, WRITE } last_action = NONE;
```

Comment:
Could you change the enum values to CamelCase?

Reply:
Done. Also changed type to `enum class`.